DataFormats/TotemRawData/interface/TotemRawEvent.h

```diff
@@ -0,0 +1,60 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*    
+****************************************************************************/
+
+#ifndef _Totem_RawEvent_h_
+#define _Totem_RawEvent_h_
+
+#include <ctime>
+#include <map>
+
+/**
+ * Encapsulates meta-data (non-VFAT data) of TOTEM raw event.
+**/
+class TotemRawEvent
```

Comment:
Raw data frame blocks don't really belong in  DataFormats.
These are intermediate data structures needed for RawToDigi or DigiToRaw and can be kept in the unpacker package(s) as transient formats.
It will still be possible to save them for some specialized analysis, but they should not be expected to be a part of standard CMS dataset outputs.